python/pyspark/sql/tests.py

```diff
@@ -4062,18 +4062,42 @@ def test_vectorized_udf_unsupported_types(self):
             with self.assertRaisesRegexp(Exception, 'Unsupported data type'):
                 df.select(f(col('map'))).collect()
 
-    def test_vectorized_udf_null_date(self):
+    def test_vectorized_udf_dates(self):
```

Comment:
shall we have a new test to directly verify the `toPandas` works?

Reply:
Maybe `ArrowTests.test_toPandas_arrow_toggle`:

https://github.com/apache/spark/blob/ebdbd8c4a06a4da52fc61b1dc98d6e2f2facdf9c/python/pyspark/sql/tests.py#L3461-L3464

?

In addition, I'll modify it to check between its expected Pandas DataFrame.